content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary_YubiHSM_2_Device.adoc

```diff
@@ -1,49 +1,76 @@
 == Configure the Primary YubiHSM 2 Device
 
-The YubiHSM Setup program is used to perform the initial configuration of the primary YubiHSM 2 device. The YubiHSM 2 is configured so that the necessary key material is generated on the YubiHSM 2 device:
+The YubiHSM Setup program is used to perform the initial configuration of the primary YubiHSM 2 device so that the necessary key material is generated on it:
 
-* One wrap key
-* One application authentication key
-* One audit key
+* One wrap key to be split as described in "Key Splitting and Key Custodians" in https://developers.yubico.com/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc[Prerequisites and Preparations].
+* One application authentication key for authenticating to the YubiHSM 2 through the KSP. This allows the KSP to perform operations in the YubiHSM 2.
 
-The wrap key is split between a predetermined number of key custodians, and each share should be recorded by its custodian. The authentication key and the audit key are exported under wrap to a file located in the current working directory.
+[Note]
```

Comment:
this should probably be NOTE to have the desired effect

Reply:
Right, it turns out the preview doesn't show everything exactly the way it will be once it has been merged. Corrected in both occurrences in this file.